config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml

```diff
@@ -9,7 +9,7 @@ spec:
       node.kubernetes.io/worker: ""
   config:
     installType: bundle
-    payloadImage: quay.io/confidential-containers/runtime-payload-ci:enclave-cc-HW-cc-kbc-latest
+    payloadImage: quay.io/confidential-containers/runtime-payload:enclave-cc-HW-cc-kbc-v0.11.0
```

Comment:
If enclave-cc is not updating to 0.12 this release, can you update the commit message that says it is please?

Reply:
Done, thanks.